brion/circuit.cpp

```diff
@@ -43,11 +43,12 @@ enum Section
     SECTION_CIRCUITSEEDS,
     SECTION_UNKNOWN
 };
+}
 
-class Circuit
+class Circuit::_Impl
```

Comment:
OK, being pedantic it's a nested types, as it seems that's the term used in the standard instead of "member type".